test/unittests/parser_test.cpp

```diff
@@ -661,15 +661,15 @@ TEST(parser, global_float)
     EXPECT_FALSE(module.globalsec[0].type.is_mutable);
     EXPECT_EQ(module.globalsec[0].type.value_type, ValType::f32);
     EXPECT_EQ(module.globalsec[0].expression.kind, ConstantExpression::Kind::Constant);
-    EXPECT_EQ(Value(module.globalsec[0].expression.value.constant).f32, 1.2f);
+    EXPECT_EQ(module.globalsec[0].expression.value.constant.f32, 1.2f);
     EXPECT_TRUE(module.globalsec[1].type.is_mutable);
     EXPECT_EQ(module.globalsec[1].type.value_type, ValType::f64);
     EXPECT_EQ(module.globalsec[1].expression.kind, ConstantExpression::Kind::Constant);
-    EXPECT_EQ(Value(module.globalsec[1].expression.value.constant).f64, 3.4);
+    EXPECT_EQ(module.globalsec[1].expression.value.constant.f64, 3.4);
     EXPECT_FALSE(module.globalsec[2].type.is_mutable);
     EXPECT_EQ(module.globalsec[2].type.value_type, ValType::f64);
     EXPECT_EQ(module.globalsec[2].expression.kind, ConstantExpression::Kind::GlobalGet);
-    EXPECT_EQ(module.globalsec[2].expression.value.constant, 1);
+    EXPECT_EQ(module.globalsec[2].expression.value.constant.f64, 1);
```

Comment:
This fails tests.

Reply:
This is an index of `GlobalGet`, it should be `.i64`

Reply:
The other cases above use `f32` and `f64`.

Reply:
Ah never mind, didn't see the `GlobalGet` vs `Constant`, will fix.

Reply:
Btw, the test below uses `value.global_index` and not `value.constant` for that case, so I think this test here is a bit wrong.

Reply:
Oh right, should be `value.global_index`

Reply:
Fixed.